### DISCUSSION TITLE
Diferencia entre UX y UI

### DISCUSSION THREAD

**Discussion by Emmanuel0307** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#75] [Q&A] [answered]
Hola compañeros, tengo una duda sobre la diferencia entre el UX y el UI, mas en especifico con la practica, en el ejercicio 4 a lo que entendi es que es tiene que realizar un Wireframe sobre la estructura que tendra nuestra pagina web, pero con el ejercico de UX la verdad es que no se como es que debo de realizarlo. Espero que me puedan aclarar esa duda que tengo.

MUCHAS GRACIAS Y BUENAS NOCHES!

**Comment by Ingrzs** [comment]
Son las interfaces del sistema , todas las ventanas... saludos

**Comment by doguedogue** [comment]
Checa estos comentarios 

https://github.com/LaunchX-InnovaccionVirtual/FrontEnd-Mision/discussions/69

**Comment by carlos-gmx3** [comment, marked as answer]
van de la mano y a la vez son diferentes
UI se refiere a lo que vez en la pagina de un sitio, lo colores las fuentes, los botones, menús etc
UX se refiere a funciones de un sitio que influyen en el sentir de un usuario con base en su experiencia usando una o más aplicaciones, te dejo unos ejemplos 
1. es muy cómodo que en outlook cuando seleccionas correos y aprietas la tecla supr la pagina hace la función de borrar los correos seleccionados a diferencia de gmail q tienes q hacer click en el botón teniendo que retirar tu mano del teclado
2. Al integrar el modo oscuro en facebook es posible estar más tiempo en la aplicación sin sentir fatiga visual debido al brillo del color blanco... pero es molesto que cada q se actualiza esta función desaparezca por unas horas
3. Muchos sitios web depende de la publicidad y es entendible. Para mi es molesto que cuando estés leyendo un articulo en internet haya anución entre parrafos en vez de poner la publicidad al principio y al final del articulo, pierdes el hilo
¿Por que van de la mano? si por ejemplo, en una pagina web informativa no se integra un formulario de contacto y se pone directo un correo personal de quien solicitó el sitio. ¿Como crees q se sienta un cliente al tener que copiar y pegar la direccion para mandar un mensaje fuera de la pagina? ¿Como se sentirá el propietario de la aplicación al notar que le llegan muchos correos de spam porque su dirección personal es pública en la web?

**Comment by CarlosSalinasRuiz** [comment]
Hola compañero, la diferencia a mi parecer es que UX es el boceto ya sea a mano o digital de la estructura que lleva la pagina o aplicación, en donde no hay imágenes, texto o colores acorde los requerimientos. Y sobre la parte UI se enfoca a previo boceto que se hizo en UX en el cual vamos a ir siendo creativos en colocar las imágenes, textos y colores.

Un ejemplo visual de como debería de quedar la practica es la siguiente:
![273820373_1549036282141018_2569304555413662374_n](https://user-images.githubusercontent.com/41224166/155331862-f74a1127-30c6-4e92-8daf-51f9525f1b95.gif)

UX:
![image](https://user-images.githubusercontent.com/41224166/155331962-acb3886d-ac53-4a1c-a3b9-003b60062f2f.png)

UI:
![image](https://user-images.githubusercontent.com/41224166/155331987-ba0b0486-6ae5-4942-acef-987b9d88adc5.png)

**Reply by AndresAVargas** [reply]
En el Ux no podemos hacer un flujo de este tipo? o esto que hice es otro cosa?
![flujo](https://user-images.githubusercontent.com/88987587/155616727-9df58c4c-cdd8-40af-9bb9-bc32e9f3c9bf.png)
o

**Reply by CarlosSalinasRuiz** [reply]
Hola Andrés, en los diseños UX y UI son bocetos de la plataforma, este diagrama que presentas es muy valido para hacer el desarrollo de dichos bocetos, pero en terminología se indica que UX es la experiencia de usuario y la UI la interfaz de usuario.

Por lo cual este diagrama se puede interpretar como el flujo del aplicativo.

**Reply by carlos-gmx3** [reply]
Puede ser parte de... pero involucra más las funciones y el acceso y la intuición con la que estás se pueden usar

**Reply by ChanLuis** [reply]
Bro, no deseches ese flujo, puedes usarlo en el diagrama de requerimientos

**Comment by Emmanuel0307** [comment]
Perfecto compañeros muchisimas gracias por sus respuestas. Fueron de mucha ayuda.
